### ISSUE TITLE
I want to create multiple schemas in prisma orm

### ISSUE THREAD

**Issue by apolanc** [second-state/chat-with-chatgpt#349]
Prisma currently only supports one single Prisma Schema file. Developers want more flexibility for managing their Prisma Schema files.

Motivations:

breaking up large schema files to make them more manageable
flexible integration with certain architectures, i.e. each module has its own self-contained logic including its schema

**Comment by alabulei1** [comment]
Hello! Prisma currently only supports one single Prisma Schema file. However, there are ways to achieve more flexibility with managing multiple schemas within Prisma.

One approach is to break up large schema files into smaller, more manageable ones. This can make it easier to work with and maintain the schema. You can split your schema into different files and then import them into your main schema file.

Another approach is to use modular architecture, where each module has its own self-contained logic, including its own schema. This can help with organization and separation of concerns. Each module can have its own Prisma Schema file, and you can use them together in your application.

You can find more information and examples on how to achieve these approaches in the Prisma documentation and the Prisma community forum.

**Comment by apolanc** [comment]
are there any other open issues in prisma repo regarding this?

**Comment by alabulei1** [comment]
Yes, there are several open issues in the Prisma GitHub repository related to managing multiple schemas. Here are a few you might find useful:

1. [Support for splitting the Prisma schema](https://github.com/prisma/prisma/issues/35): This issue discusses the possibility of splitting the Prisma schema into multiple files.

2. [Support for multiple Prisma schema files](https://github.com/prisma/prisma/issues/1130): This issue suggests allowing multiple Prisma schema files to provide more flexibility in managing schemas.

3. [Capability to use multiple Prisma schemas](https://github.com/prisma/prisma/issues/2829): This issue proposes providing the capability to use multiple Prisma schemas side by side.

You can find more related issues by searching the Prisma GitHub repository using keywords like "multiple schema" or "split schema". It's a good idea to check these issues periodically for any updates and to contribute your thoughts and suggestions if necessary.
